draft-ietf-scitt-architecture.md

```diff
@@ -619,6 +619,40 @@ For example, consider a Transparency Service implemented using a set of replicas
 Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
 This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
 
+### Configuration
```

Comment:
We must state something about a default configuration that brings up the `append only log` managed by the Transparency Service. Once the Service is up and running the default configuration can be modified.

Reply:
This comment is ONLY applicable if-we stick to the original definition of the `Configuration` which I strongly believe we should avoid.

Reply:
I was also struggling with the text that referenced ordering. This sounds implementation-specific. I wonder if we should simply say the Transparency Service MUST have a registration and configuration persisted as signed statements. Should how the service is "booted up" be outside the specification?

Reply:
The default configuration before registering the first configuration is entirely implementation specific (it just needs to explain how the first explicit configuration is authorized and registered) so I don't think there is much to say about it.